doc/source/ray-air/examples/gptj_deepspeed_fine_tuning.ipynb

```diff
@@ -414,10 +414,10 @@
     "🤗 Transformers and Ray AIR's integration (`HuggingFaceTrainer`) allow you to easily configure and use DDP and DeepSpeed. All you need to do is specify the DeepSpeed configuration in the [`TrainingArguments`](https://huggingface.co/docs/transformers/en/main_classes/trainer#transformers.TrainingArguments) object.\n",
     "\n",
     "```{tip}\n",
-    "There are many DeepSpeed settings, allowing you to trade-off speed for memory usage. The settings used below are tailored to the cluster setup used (16 g4dn.4xlarge nodes) and batch size of 16. Some things to keep in mind:\n",
-    "- If your GPUs support bfloat16, it is recommended to use that instead of float16 mixed precision as it gives better performance and prevents overflows. Simply replace `fp16=True` with `bf16=True` in `TrainingArguments`\n",
+    "There are many DeepSpeed settings, allowing you to trade-off speed for memory usage. The settings used below are tailored to the cluster setup used (16 g4dn.4xlarge nodes) and per device batch size of 16. Some things to keep in mind:\n",
+    "- If your GPUs support bfloat16, it is recommended to use that instead of float16 mixed precision as it gives better performance and prevents overflows. Simply replace `fp16=True` with `bf16=True` in `TrainingArguments`.\n",
```

Comment:
```suggestion
    "- If your GPUs support bfloat16, use that instead of float16 mixed precision to get better performance and prevent overflows. Replace `fp16=True` with `bf16=True` in `TrainingArguments`.\n",
```